Deprecated Kutztown University locality record

stepps00 opened the pull request:
Fixes #360.

* Deprecated locality record of Kutztown University. Did not add in a `superseded_by` value because the correct Kutztown University record is a venue point.
* Updated `lastmodified` date.

nvkelso commented:
Should we elevate the venue to a campus, and preserve it's polygon geometry?

stepps00 commented:
This PR deprecated the Kutztown locality record and superseded it to a new campus record that was created in a [PR](https://github.com/whosonfirst-data/whosonfirst-data-venue-us-pa/pull/2) in the `whosonfirst-data-venue-us-pa` repo.

Since this placetype change qualifies as a "significant event," both existing records were deprecated and superseded to a new campus record... and because these records are located in separate repos, we have two separate issues. 

[WOF-data issue](https://github.com/whosonfirst/whosonfirst-data/issues/360) and [WOF-venue issue](https://github.com/whosonfirst-data/whosonfirst-data-venue-us-pa/issues/1).

nvkelso commented on data/101/719/267/101719267.geojson:
```diff
@@ -14,6 +14,7 @@
     "mps:latitude":40.51186,
     "mps:longitude":-75.788583,
     "mz:hierarchy_label":1,
+    "mz:is_current":1,
```
How do I read `"mz:is_current":1`?  As currently "closed" (since `edtf:cessation` has been set)?

/cc @thisisaaronland

stepps00 replied:
After looking over other records ([1](https://raw.githubusercontent.com/whosonfirst-data/whosonfirst-data/9a51c5d7150b7262a1728805b8fb749fd36e827e/data/102/449/711/5/1024497115.geojson), [2](https://raw.githubusercontent.com/whosonfirst-data/whosonfirst-data/a46a58c00f65d6b356440be03bdfd64ad999c85d/data/858/655/85/85865585.geojson), [3](https://raw.githubusercontent.com/whosonfirst-data/whosonfirst-data/a46a58c00f65d6b356440be03bdfd64ad999c85d/data/421/180/189/421180189.geojson)), it looks like we're setting `mz:is_current` to `1` when the record is current and setting the value to `0` when the record is no longer valid.

Will update this record when I confirm this is true.

nvkelso commented:
Besides my comment about [mz:is_current](https://github.com/whosonfirst-data/whosonfirst-data/pull/363/files#r76684466), we're ready to go.

stepps00 commented:
Verified that `mz:is_current` should be set to `0` when a feature is no longer valid (also updated [properties repo](https://github.com/whosonfirst/whosonfirst-properties/commit/6a172acd12acbba5fa1913379b877668d18785f5)). Updated the record in this PR with correct value.

Over to @thisisaaronland for import.
